components/icon/__tests__/icon.dom-snapshot-test.jsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-env jest */
```

Comment:
Please use Storybook Storyshots for DOM snapshots of stories instead of tests in component folder.

Reply:
Which you appear to already have up in `components/combobox/__docs__/__snapshots__/storybook-stories.storyshot`

Reply:
sure, yes I added both. I will remove icon.dom-snapshot-test.jsx then.

Reply:
I thought I did the correct thing by removing the extra comma in components/icon/index.jsx, I will add it back. Though, lint passes on local.